lessons/operator/03/README.md

```diff
@@ -375,208 +217,446 @@ $ sensuctl configure -n \
+> **PROTIP:** If you like to automate things like this using shell scripts, you might already be thinking about how to parse the output of the `sensuctl api-key grant` command.
+>
+> The following example should do the trick for Mac and Linux users:
+>
+> **Example:** Parsing an API key using `awk`
+> ```shell
+> export SENSU_API_KEY=$(sensuctl api-key grant sensu | awk -F "/" '{print $NF}')
+> ```
 
-1. **Get information about a specific entity using the `sensuctl entity info` command.**
+## Listing and Viewing Resources
 
-   ```shell
-   sensuctl entity info learn.sensu.io
-   ```
+One of the common uses of `sensuctl` is to list and view resources like namespaces, users, and entities. 
 
-1. **Try exploring some other resources.**
+### EXERCISE 4: List Sensu Resources
+#### Scenario
 
-   _NOTE: don't forget to use `--help`; for example, `sensuctl --help` will output a list of "management commands" which are effectively API resources that are accessible via `sensuctl`._
+You want to explore the resources available in Sensu. 
 
-**NEXT:** when you're done exploring `sensuctl` resource list and info commands, you're ready to move on to the next step.
+#### Solution
 
-## EXERCISE 6: create an API Key for personal use
+To see a list, use the `sensuctl <resource_type> list` command pattern. 
+The `list` subcommand is available for nearly all resources, including common resources like namespaces, users, and entities.
 
-1. **Use the `sensuctl api-key grant` command to create an API Key.**
+#### Steps
 
-   Grant an api-key for the default user (`sensu`).
+1. **Use the `sensuctl namespace list` Command to Get a List of Namespaces.**
 
    ```shell
-   sensuctl api-key grant sensu
+   sensuctl namespace list
    ```
-
-   The output of this command will look like:
+   
+   **Example Output:**
+   ```shell
+      Name
+    ─────────
+     default
+     trainee
+   ```
+1. **Use the `sensuctl user list` Command to Get a List of Users.**
 
    ```shell
-   Created: /api/core/v2/apikeys/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+   sensuctl user list
    ```
+   
+   **Example Output:**
+   ```shell
+     Username           Groups            Enabled
+    ────────── ───────────────────────── ─────────
+     agent      system:agents             true
+     sensu      cluster-admins            true
+     trainee    trainee,trainee:trainee   true
+   ```
+>**PROTIP:** Want to explore what `sensuctl` can do? Try using the  `--help` option to show a list of commands. This option is available on all commands and subcommands.
 
-1. **Save the API Key for use in a future exercise.**
+## Inventory Management
 
-   For the purposes of this workshop, we want to capture this API key (the
-   `xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx` part of the output) for use in a future exercise.
-   You can either copy the output from the `sensuctl api-key grant` command manually, like this:
+One common use case for `sensuctl` is to manage a real-time "inventory of nodes".
```

Comment:
Makes sense! Although I will say that while I normally strongly encourage using the same terms everywhere, very consistently, there is an issue of voice. Many terms that primarily make sense in a marketing context don't sit well in a developer-focused/documentation context. They stand out as "buzzwordy" and lack definite meaning. See my commentary on this in the original review document. If possible we want to talk about things that are tangible and actionable and use the same terms that the tools use to describe them. There is no `sensuctl manage <endpoint>` command, so what does that translate to? Here, it translates to `sensuctl entities list`.. So rather than saying Sensu "manages endpoints" we could say it "lists entities" to be more rooted in the developer mindset. 

However, "listing entities" is also not well phrased to imply its use for endpoint management. This is a product issue that is larger than this individual document or workshop. Instead of calling them "entities" maybe we should call them "endpoints" in the code/events ... and instead of saying Sensu can "manage endpoints" we could say it can "show an inventory of endpoints" in the marketing copy.  

Then the command would look like `sensuctl endpoint list` and the marking copy would say `show an inventory of endpoints", which both use the same nouns, and *almost* the same verbs. The verb "manage" implies something well beyond inventorying to me.. Like creating new ones, starting/stopping existing one, changing endpoint configurations, etc. Sensu doesn't do all of that natively. So does the marketing copy really align with Sensu's capabilities? The gap there is revealed in situations like this. It's also unfortunately common for marketing speak to imply or suggest that something can do more than it actually does, but developer docs cannot afford the luxury of hyperbole. They must be grounded in the actual capabilities and terminology.